src/components/project-overview/project-overview.tsx

```diff
@@ -27,5 +28,13 @@ export default async function ProjectOverview() {
         </Fragment>
       ))}
     </Box>
+  ) : (
+    <Box className="project-overview-container">
+      <Box className="project-overview-card">
+        <Card>
+          <h3>Github API</h3>
+        </Card>
+      </Box>
+    </Box>
```

Comment:
Let's return null here now, and comment to revisit this part when Github API call is ready

Reply:
Done